server/routes/api.js

```diff
@@ -11,7 +11,7 @@ router.post('/test', (req, res) => {
 
 router.post('/neighborhoodList', (req, res) => {
     console.log(req.body)
-    var jsonString = JSON.stringify(operations.getNeighborhoodList());
+    var jsonString = JSON.stringify(new OperationsLayer());
```

Comment:
Does it make sense to construct a new OperationsLayer() in this route every time it is called? What if we made a function in the operations.js that constructs a new operationsLayer if isLoaded is false, but uses the existing OperationsLayer class that we can construct in the operations.js? Either way, I can see that the current implementation doesn't create more than on neighboorhoodList, so the current implementation is fine.

Reply:
Technically, I think it would be preferable to simply call `OperationsLayer.getNeighborhoodList()`.